fs-repo-migrations/main.go

```diff
@@ -82,26 +98,45 @@ func main() {
 		os.Exit(1)
 	}
 
-	if vnum > *target && !*revertOk {
+	if vnum > target && !*revertOk {
 		fmt.Fprintln(os.Stderr, "ipfs migration: attempt to run backward migration\nTo allow, run this command again with --revert-ok")
 		os.Exit(1)
 	}
 
-	if vnum == *target {
-		fmt.Fprintln(os.Stderr, "ipfs migration: already at target version number")
+	if vnum == target {
+		fmt.Fprintln(os.Stderr, "ipfs migration: already at version", target)
 		return
 	}
 
 	ipfsDir, _ := migrations.IpfsDir("")
 	fmt.Printf("Found fs-repo version %d at %s\n", vnum, ipfsDir)
-	prompt := fmt.Sprintf("Do you want to upgrade this to version %d? [y/n]", *target)
+	prompt := fmt.Sprintf("Do you want to upgrade this to version %d? [y/n]", target)
 	if !(*yes || yesNoPrompt(prompt)) {
 		os.Exit(1)
 	}
 
-	err = migrations.RunMigration(context.Background(), fetcher, *target, "", *revertOk)
+	err = migrations.RunMigration(context.Background(), fetcher, target, "", *revertOk)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, "ipfs migration: ", err)
 		os.Exit(1)
 	}
 }
+
+func latestRepoMigration(fetcher migrations.Fetcher) (int, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	var latestMigration int
+	var err error
+	for i := currentVersion; err == nil; i++ {
```

Comment:
This is fine for now, but let's file an issue and drop a TODO to remove `currentVersion`. Right now it just saves us a few HTTP queries so why not, but it's also totally doable to just grab the list of all `fs-repo-*-to-*` and calculate it at once without multiple requests.

Reply:
Done.  Added TODO.